frontend_tests/puppeteer_lib/common.js

```diff
@@ -435,7 +435,10 @@ class CommonUtils {
         await page.click(organization_settings);
         await page.waitForSelector("#settings_overlay_container.show", {visible: true});
 
-        const url = page.url();
+        // `page.url()` does not show url fragment when running with Firefox.
+        // Here is the link to the issue https://github.com/puppeteer/puppeteer/issues/6787.
+        // TODO: When the above issue is solved please replace the below line with `page.url()`.
+        const url = await page.evaluate("location.href");
```

Comment:
Can we extract this to a `common.page_url()` method we call from 3 places?  Would save on copy/pasting the comment.

Reply:
@timabbott I suggest we name the function `common.page_url_with_fragment()` because `page.url()` works fine when we don't need URL fragment.

Reply:
@timabbott I suggest we name the function `common.page_url_with_fragment()` because `page.url()` works fine when we don't need URL fragment.

Reply:
Yeah, that works.